lib/createConfig.js

```diff
@@ -1,7 +1,28 @@
 const { merge } = require('webpack-merge');
+const path = require('path');
 const getBaseConfig = require('./getBaseConfig');
+const presets = require('./presets');
 
 module.exports = (commandName, configFragment = {}) => {
   const baseConfig = getBaseConfig(commandName);
+
+  if (commandName === 'webpack-prod') {
+    const cacheFolderPath = presets[commandName].getMfePath();
+    baseConfig.cache = {
```

Comment:
If consumers want to change the settings for `cache` for their specific MFE only, this approach of setting `cache` in `createConfig` limits its configurability compared to relying on adding it to the default `webpack.prod.config.js` file instead.

Reply:
We could pass this config as a default and pass a cache configuration in `configFragment` or as a third param, couldn't we?

Reply:
We could, but it would unnecessary if the `cache` property were added to the base `config/webpack.prod.config.js` file directly :) Once added there, consumers should be able to override/customize it via `createConfig` already.